### ISSUE TITLE
Tabs don't persist after app restart

### ISSUE THREAD

**Issue by ergl** [MacPass/MacPass#962]
**Describe the bug**
MacPass won't remember it had several database tabs open after restarting the app.

**To Reproduce**
1. Open MacPass
2. Open two password databases, each will create a new tab.
3. Quit the app, with Cmd+Q.
4. Re-open MacPass
5. Only the database on the first tab will be visible.

Also more details: The order of the tabs doesn't matter, only the order in which you opened the databases inside MacPass: If, after step 2, you drag the second tab so that it becomes the first one, on step 5, only the original database will be present.

**Expected behavior**
Both tabs should still be visible.

**Desktop (please complete the following information):**
 - macOS: Mojave 10.14.5
 - Version: 0.7.9 (21570)

**Additional context**
A previous issue (#885) brought this up originally, but I figured it would be nice to have a dedicated issue for this. It was noted quitting the app through "Force Quit" solved the issue. I could reproduce that behaviour one time, but never since. I imagine it is a non-deterministic thing.

**Comment by mstarke** [comment]
MacPass stores only the last database that was opened in a proprietary format. It otherwise depends on macOS for restoration. Do you habe close all Windows when applications quit set in you general system preferences? This immigrier prevent the tabs from being re-opened.

**Comment by mstarke** [comment]
To clarify the behavior a bit. MacPass remebers the database when a window is closed. Closing the window with tabs will do the same as macOS treats each tab as a separate window and manages the tabs ansparently. So if you close a database the last closed window which seems to be in reverse order of opening will set the last database and thus be re-opened in re-launch.

**Comment by ergl** [comment]
> Do you have close all windows when quitting an applications set in your general system preferences?

I didn't know about this feature, and apparently I do have it selected. Thanks! Turning it off solves this issue.

Perhaps this is also what is happening with #885?
